client/js/src/main/scala/zio/zmx/client/frontend/state/Command.scala

```diff
@@ -128,7 +125,7 @@ object Command {
         }
       )
       cfg match {
-        case cfg: PanelConfig.DisplayConfig =>
+        case cfg: PanelConfig.DisplayConfig if cfg.metrics.nonEmpty =>
```

Comment:
Keep the centred + (Plus) button if the user didn't add any metrics when closing the Configuration modal dialog by clicking the `Apply` button.

Reply:
Actually this is only part of the effort.

Reply:
Hmm ... We would ned to make the panel an empty panel if after closing the dialog no metrics are selected.

Reply:
Let me test different scenarios a bit more.

Reply:
Reverted this change.